docpages/example_programs/the_basics/using_cache.md

```diff
@@ -0,0 +1,18 @@
+\page using-cache Using Cache
+
+Sometimes you may need information that is not directly available in the event callback object.
+
+To handle this DPP maintains a cache of commonly used data for you.
+
+@note As of August 30th, 2022, Discord made Guild Members a privileged intent. You must have GUILD_MEMBERS intent enabled for your app from discord developers portal to be able to look for members in cache.
```

Comment:
this isn't technically true, id you don't have the intent the cache will "trickle fill" slowly with members as the bot sees them, e.g. when they send a message, join a VC, or send a slash command. you might also want to mention how slash commands have resolved sets that eliminates the need to look in the cache for data.